### PR TITLE
CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by gaushikmr**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at . All
```
Comment (silentsoft) [inline CODE_OF_CONDUCT.md:58]: The original `CODE_OF_CONDUCT` requires email address to the end of `~~~~ contacting the project team at <email>.` in `Enforcement` section. or.. how about remove the word ` at ` and just end `~~~~ project team.` ?

Comment (gaushikmr) [reply CODE_OF_CONDUCT.md]: @silentsoft Done can u check

Comment (silentsoft) [reply CODE_OF_CONDUCT.md]: @gaushikmr By the way, did you really discuss this with maintainers before open a pull request ?

Comment (gaushikmr) [reply CODE_OF_CONDUCT.md]: @No

**Pull request closed without merging**